<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateGroupDlg</class>
 <widget class="QDialog" name="CreateGroupDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create New Group</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Choose parent group: </string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTreeWidget" name="twGroups">
     <attribute name="headerVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Enter the new group name:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="lneGroupName"/>
     </item>
     <item>
      <widget class="QPushButton" name="btnAddGroup">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CreateGroupDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>222</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CreateGroupDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>228</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnAddGroup</sender>
   <signal>clicked()</signal>
   <receiver>CreateGroupDlg</receiver>
   <slot>addGroup()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>313</x>
     <y>180</y>
    </hint>
    <hint type="destinationlabel">
     <x>361</x>
     <y>166</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lneGroupName</sender>
   <signal>textChanged(QString)</signal>
   <receiver>CreateGroupDlg</receiver>
   <slot>checkGroupName()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>149</x>
     <y>188</y>
    </hint>
    <hint type="destinationlabel">
     <x>356</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>twGroups</sender>
   <signal>currentItemChanged(QTreeWidgetItem*,QTreeWidgetItem*)</signal>
   <receiver>CreateGroupDlg</receiver>
   <slot>checkGroupName()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>94</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>360</x>
     <y>72</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>checkGroupName()</slot>
  <slot>addGroup()</slot>
 </slots>
</ui>
